app/presenters/result_set_presenter.rb

```diff
@@ -50,6 +50,10 @@ def sort_option
     presenter[:options].find { |o| o[:selected] }
   end
 
+  def has_sort_options
```

Comment:
What does this method name mean?

Reply:
Ah pk, after reading your response to my other comment, this makes more sense. It would really help if you add this reasoning to the commit message.

Reply:
Thanks, yep that would be helpful. I've now updated the What section to try and better describe the issue.